rfc/github-cdevents.md

```diff
@@ -0,0 +1,37 @@
+| pull_request                | dev.cdevents.change.updated                     | This event occurs when there is activity on a pull request. <details><summary>Actions</summary> {"Action type":{"assigned", "auto_merge_disabled", "auto_merge_enabled", "closed", "converted_to_draft", "demilestoned", "dequeued", "enqueued", <br/>"archived", "labeled", "locked", "milestoned", "ready_for_review","reopened", "review_request_removed", "review_requested", "synchronized", "unassigned", "unlabeled", "unlocked"}</br>                    |
+| pull_request_review_comment | dev.cdevents.change.reviewed                    | This event occurs when there is activity on a pull request review comment. <details><summary>Actions</summary> {"Action type":{"created", "reviewed", "commented"}                                                                                                                                                                                                                                                                                               |
+| pull_request_review         | dev.cdevents.change.reviewed                    | This event occurs when there is activity relating to a pull request review. <details><summary>Actions</summary> {"Action type":{"edited", "dismissed", "submitted"}  </details>                                                                                                                                                                                                                                                                                  |
+| pull_request_review_thread  | dev.cdevents.change.reviewed                    | This event occurs when there is activity relating to a comment thread on a pull request. <details><summary>Actions</summary> {"Action type":{"resolved", "unresolved"}  </details>                                                                                                                                                                                                                                                                               |
+| package/register_package    | dev.cdevents.artifact.modified/registry_package | This event occurs when there is activity relating to GitHub Packages/registry. <details><summary>Actions</summary> {"Action type":{"updated"}  </details>                                                                                                                                                                                                                                                                                           |
```

Comment:
We do not have an artifact.modified. Is the idea to introduce this to the spec? Basically, this would be the same published or packaged event.

I see @afrittoli has the same comment earlier with a few others to map to. I think this one just got missed

Reply:
i will remove for the moment.